pkgs/applications/science/misc/openmvs/default.nix

```diff
@@ -18,12 +38,34 @@ stdenv.mkDerivation rec {
   };
 
   # SSE is enabled by default
-  cmakeFlags = lib.optional (!stdenv.isx86_64) "-DOpenMVS_USE_SSE=OFF";
+  cmakeFlags = [
+    "-DOpenMVS_ENABLE_TESTS=OFF"
+  ] ++ lib.optional (!stdenv.isx86_64) "-DOpenMVS_USE_SSE=OFF";
 
-  buildInputs = [ eigen opencv cgal ceres-solver vcg glfw boostWithZstd ];
+  buildInputs = [
+    boostWithZstd
+    breakpad
+    ceres-solver
+    cgal
+    eigen
+    glfw
+    gmp
+    libjpeg
+    libpng
+    libtiff
+    mpfr
+    opencv
+    openmp
+    vcg
+  ];
 
   nativeBuildInputs = [ cmake pkg-config ];
 
+  postFixup = ''
```

Comment:
this should be postInstall. The default fixup phase has some logic that depends on the location of files, so it's better if they are already at their right place.